dask_bigquery/tests/test_core.py

```diff
@@ -148,7 +148,7 @@ def write_dataset(google_creds):
 
     dataset_id = f"{sys.platform}_{uuid.uuid4().hex}"
 
-    yield google_creds, project_id, dataset_id
+    yield google_creds, project_id, dataset_id, None
```

Comment:
Just for my understanding, why do we need the `None` here? Is this because in this case there is no table_id? If so can we add a comment that mentions that?

Reply:
Yes, that's why, just so `write_existing_dataset` and `write_dataset` have the same signature.